mods/voidmulti.sc2mod/base.sc2data/gamedata/weapondata.xml

```diff
@@ -112,9 +117,16 @@
         <MinScanRange value="5.5"/>
         <TargetFilters value="Ground,Visible;Missile,Stasis,Dead,Hidden,Invulnerable"/>
     </CWeaponLegacy>
+    <CWeaponStrafe id="InterceptorBeam">
+        <LoiterInnerRadius value="3"/>
+        <LoiterRadius value="5"/>
+    </CWeaponStrafe>
```

Comment:
Interceptor flying radius around target increased
- LoiterInnerRadius increased from 2 to 3
- LoiterRadius increased from 4 to 5

Does this have DPS implications since the Interceptors are flying further?